### DISCUSSION TITLE
Do I need to resize images to 32x32 to use the FAST text detector?

### DISCUSSION THREAD

**Discussion by vedantroy** [mindee/doctr#1663] [Q&A]
Looking at the sample code for TextNet:
```
>>> import tensorflow as tf
>>> from doctr.models import textnet_base
>>> model = textnet_base(pretrained=False)
>>> input_tensor = tf.random.uniform(shape=[1, 32, 32, 3], maxval=1, dtype=tf.float32)
>>> out = model(input_tensor)
```

It looks like the input shape is 32x32. Does this mean I need to resize the image to 32x32 pixels? I worry that's too low resolution to detect text.

**Comment by felixdittrich92** [comment]
Hi @vedantroy :wave:,

`textnet` models are the backbones for the `fast` models which are pretrained on character classification (32x32 images).

Best regards,
Felix
